Azure/azure-cli
# [Backup] `az backup restore restore-disks`: Allow `--disk-encryption-set-id` for cross region restore

**Related command**

`az backup restore restore-disks`

**Description**

Allows the usage of the `--disk-encryption-set-id` parameter for the cross region restore scenario (Can use it alongside the `--use-secondary-region` flag).

**Testing Guide**

Run a `restore-disks` command with the `--disk-encryption-set-id` and `--use-secondary-region` parameters both. The job should be triggered. Earlier, there would have been an error message saying: "disk_encryption_set_id can't be specified".

**History Notes**


---

This checklist is used to make sure that common guidelines for a pull request are followed.
